src/python/grpcio_tests/tests/observability/_observability_test.py

```diff
@@ -413,6 +445,41 @@ def _validate_spans(
             self.assertTrue(prefix_exist)
 
 
+def unary_unary_call(port, metadata=None):
+    with grpc.insecure_channel(f"localhost:{port}") as channel:
+        multi_callable = channel.unary_unary(_UNARY_UNARY)
+        if metadata:
```

Comment:
Nit: You don't need the conditional. You can just do `metadata = metadata or ()`